app/lib/test_applications.rb

```diff
@@ -340,10 +340,10 @@ def confirm_offer_conditions(choice)
       fast_forward
       ConfirmOfferConditions.new(actor: actor, application_choice: choice).save
       choice.update_columns(recruited_at: time, updated_at: time)
+      # service generates two audit writes, one for status, one for timestamp
+      choice.audits.second_to_last&.update_columns(created_at: time)
+      choice.audits.last&.update_columns(created_at: time)
```

Comment:
Does it matter that we're `as_provider_user` when calling `update_columns`?

Reply:
No, don't think so, update_columns bypasses the audit log.